makedebian.pl

```diff
@@ -61,7 +60,6 @@
 chmod(0755, $tmp_dir);
 mkdir($debian_dir, 0755);
 system("mkdir -p $pam_dir");
-system("mkdir -p $init_dir");
```

Comment:
I assume the plan here is to not include the init script in the RPM at all, and instead create it dynamically in atboot.pl ? If so, how will this behave when a user upgrades?

Reply:
Yes, this is the plan. If user upgrades it will be clear by the package. We don't need to clean it manually.

Reply:
Got it! Yeah, that makes sense ..